lib/svtplay_dl/service/dplay.py

```diff
@@ -139,7 +139,7 @@ def find_all_episodes(self, config):
         self._getpackages()
 
         urllocal = ""
-        if self.domain in ["dplay.dk", "dplay.no"]:
+        if self.domain in ["dplay.dk", "dplay.no", "discoveryplus.no", "discoveryplus.dk"]:
```

Comment:
remove the old domains here, they are not in use anymore.

Reply:
fixed